README.md

```diff
@@ -58,10 +58,53 @@ build/$ ctest . --output-on-failure
 
 The bci.config file is used by our internal validation environment, please just ignore it.
 
+### Build with dilithium support
+
+Dilithium is an optional feature provided by MoCOCrW. To enable the feature,
+replace the CMake invocation with:
+```
+build/$ cmake -DBUILD_TESTING=True -DDILITHIUM_ENABLED=ON ..
+```
+
+Make sure that the adapted version of libdilithium can be found by the linker.
+
+HSM and dilithium support can be enabled independently.
+
+#### Dilithium Adaptions
+
+It is not possible to take the bare dilithium implementation. The dilithium implementation was
+adapted. A new function for retrieving the public key from a private key was added. This change is
+required to compile MoCOCrW with dilithium support.
+
+There are two PRs created for dilithium. These can be found in dilithium's [github
+repository](https://github.com/pq-crystals/dilithium/).
+
+The first [PR](https://github.com/pq-crystals/dilithium/pull/68) contains the adaptions for
+retreiving the public key from a private key and is **required** in order to compile MoCOCrW with
+dilithium support.
+
+The second [PR](https://github.com/pq-crystals/dilithium/pull/69) improves the CMake file so that
```

Comment:
So the user has to download these two separate PR and merge them to the main branch locally and then compile? Which commit of the main branch?

Reply:
Done